backend/python/app/services/implementations/story_service.py

```diff
@@ -348,7 +349,29 @@ def get_story_translations(
                 .join("translation_contents")
                 .filter(*filters)
                 .filter(StoryTranslationAll.is_deleted == False)
-                .order_by(Story.id)
+                .order_by(
```

Comment:
nit:
```
func.greatest(
                            func.coalesce(
                                StoryTranslationAll.translator_last_activity, 0
                            ),
                            func.coalesce(
                                StoryTranslationAll.reviewer_last_activity, 0
                            ),
                        )
```
noticed this block is repeated, could this be saved to a variable? and then this block gets changed to:

```
if last_activity_ascending is None
else (
  my_var.asc()
  if last_activity_ascending
  else my_var.desc()
)
```